packages/fxa-auth-server/fxa-oauth-server/test/routes/token.js

```diff
@@ -21,7 +21,8 @@ const CLIENT_SECRET =
 const CLIENT_ID = '98e6508e88680e1b';
 const CODE = 'df6dcfe7bf6b54a65db5742cbcdce5c0a84a5da81a0bb6bdf5fc793eef041fc6';
 const PKCE_CODE_VERIFIER = 'au3dqDz2dOB0_vSikXCUf4S8Gc-37dL-F7sGxtxpR3R';
-const DISABLED_CLIENT_ID = 'd15ab1edd15ab1ed';
+const DISABLED_CLIENT_ID = '38a6b9b3a65a1871';
+const NON_DISABLED_CLIENT_ID = '98e6508e88680e1a';
```

Comment:
Thanks to the refactoring of the `authenticateClient` function, we now have to use real client ids in this test.